packages/mui-material/src/Grid2/Grid2.tsx

```diff
@@ -251,4 +252,16 @@ Grid2.propTypes /* remove-proptypes */ = {
   wrap: PropTypes.oneOf(['nowrap', 'wrap-reverse', 'wrap']),
 } as any;
 
+if (process.env.NODE_ENV !== 'production') {
+  const Component = Grid2 as any;
+  const requireProp = requirePropFactory('Grid2', Component);
+  // eslint-disable-next-line no-useless-concat
+  Component['propTypes' + ''] = {
+    // eslint-disable-next-line react/forbid-foreign-prop-types
+    ...Component.propTypes,
+    direction: requireProp('container'),
+    spacing: requireProp('container'),
```

Comment:
```suggestion
    spacing: requireProp('container'),
    wrap: requireProp('container'),
```

We are missing the wrap prop here.